pretty_midi/pretty_midi.py

```diff
@@ -1258,6 +1353,180 @@ def __repr__(self):
                 'time={:f})'.format(self.number, self.value, self.time))
 
 
+class TimeSignature(object):
+    """Containts the time signature numerator, denominator and
+    the event time in seconds
+
+    Attributes
+    ----------
+        numerator : int
+            Numerator of time signature
+        denominator : int
+            Denominator of time signature
+        time : float
+            Time of event in seconds
+
+    Examples
+    --------
+    Instantiate a TimeSignature object with 6/8 time signature at 3.14 seconds
+    >>> ts = TimeSignature(6, 8, 3.14)
+    >>> print ts
+    6/8 at 3.14 seconds
+
+    """
+
+    def __init__(self, numerator, denominator, time):
+        assert isinstance(numerator, int), \
```

Comment:
We can be more strict here, right?  I.e. the numerator and denominator should greater than 0.